webviz_subsurface/plugins/_tornado_plotter_fmu/_plugin.py

```diff
@@ -0,0 +1,486 @@
+            ),
+        )
+        @callback_typecheck
+        def _calc_tornado_table(
+            reference: str,
+            scale: str,
+            plot_options: List[str],
+            data: Union[str, bytes, bytearray],
+            sens_filter: Union[List[str], str],
+        ) -> Tuple[dict, dict, dict]:
+            if not data:
+                raise PreventUpdate
+            plot_options = plot_options if plot_options else []
+            data = json.loads(data)
+            if not isinstance(sens_filter, List):
+                sens_filter = [sens_filter]
+            if not isinstance(data, dict):
+                raise PreventUpdate
+            values = pd.DataFrame(data["data"], columns=["REAL", "VALUE"])
+            realizations = self._design_matrix_df.loc[
+                self._design_matrix_df["ENSEMBLE"] == data["ENSEMBLE"]
+            ]
+
+            design_and_responses = pd.merge(values, realizations, on="REAL")
+            if sens_filter is not None:
+                if reference not in sens_filter:
+                    sens_filter.append(reference)
+                design_and_responses = design_and_responses.loc[
+                    design_and_responses["SENSNAME"].isin(sens_filter)
+                ]
+            tornado_data = TornadoData(
+                dframe=design_and_responses,
+                response_name=data.get("response_name"),
+                reference=reference,
+                scale="Percentage" if scale == "Relative value (%)" else "Absolute",
+                cutbyref="Remove sensitivites with no impact" in plot_options,
+            )
+            tornado_table = TornadoTable(tornado_data=tornado_data)
+            return (
+                tornado_table.as_plotly_table,
+                tornado_table.columns,
+            )
+
+        @callback(
+            Output(
+                self.get_store_unique_id(PluginIds.Stores.DataStores.TORNADO_DATA),
+                "data",
+            ),
+            Input(
+                self.shared_settings_group(PluginIds.SharedSettings.SELECTORS)
+                .component_unique_id(Selectors.IDs.RESPONSE)
+                .to_string(),
+                "value",
+            ),
+            Input(
+                {
+                    "id": self.filters.single_filter_id,
+                    "name": ALL,
+                    "type": "single_filter",
+                },
+                "value",
+            ),
+            Input(
+                {
+                    "id": self.filters.multi_filter_id,
+                    "name": ALL,
+                    "type": "multi_filter",
+                },
+                "value",
+            ),
+        )
+        @callback_typecheck
+        def _update_tornado_with_response_values(
+            response: str, single_filters: list, multi_filters: list
+        ) -> str:
+            """Returns a json dump for the tornado plot with the response values per realization"""
```

Comment:
Improve naming - as the output of callback is the store data, which is for both plot and table view, the naming can be `_update_tornado_data_with_response_values` and comment can be `"Returns a json dump for the tornado data with the response values per realization"`